### DISCUSSION TITLE
[Bug] Resizing jumps from rem back to px

### DISCUSSION THREAD

**Discussion by rivedive** [webstudio-is/webstudio-community#31] [Q&A]
if i set my padding spacing to a rem number and hold and press the option key on my mac to use the mouse to give the same value to the opposite site the value jumps back to px.

**Comment by kof** [comment]
Reproduced, it's a bug, thanks for the report!

**Comment by rpominov** [comment]
I'm probably missing something, but I can't reproduce:


https://github.com/webstudio-is/webstudio/assets/825702/732be5d2-0e2f-4084-9ef7-98f3548cad45

**Comment by kof** [comment]
It's weird, I can't reproduce it now any more @rivedive any chance you could record a video or give us detailed step by step?

**Reply by rivedive** [reply]
funny enough i can't reproduce it today.. so i guess this one is solved for now ;). What i did was use the dropdown and not typing it in. But i tried both today and it worked as it should. I am using arc browser btw.

**Reply by rpominov** [reply]
Reproduced. Will fix.

https://github.com/webstudio-is/webstudio/assets/825702/847dde65-4fc2-46ab-8047-b3da0ab08d98

**Reply by kof** [reply]
ah so the scrub was done on the input field, not on paddin/margin UI

**Reply by rpominov** [reply]
Yeah, probably that was the catch. I can only reproduce on the input

**Comment by kof** [comment]
fixed
